extensions/panache/rest-data-panache/deployment/src/main/java/io/quarkus/rest/data/panache/deployment/properties/ResourcePropertiesBuildItem.java

```diff
@@ -2,7 +2,7 @@
 
 import io.quarkus.builder.item.MultiBuildItem;
 
-public class ResourcePropertiesBuildItem extends MultiBuildItem {
+public final class ResourcePropertiesBuildItem extends MultiBuildItem {
```

Comment:
I am wondering how this worked before...

Reply:
No idea :/